mithril-common/Cargo.toml

```diff
@@ -33,8 +33,12 @@ kes-summed-ed25519 = { version = "0.2.1", features = [
     "serde_enabled",
     "sk_clone_enabled",
 ] }
+minicbor = { version = "0.20", features = ["std", "half", "derive"] }
```

Comment:
Can this dependency be optional since it's only used behind the `fs` flag ?

Reply:
for sure, done.